test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetSourcesCommandTest.cs

```diff
@@ -366,12 +367,13 @@ public void TestVerbosityQuiet_DoesNotShowInfoMessages()
                     "-Verbosity",
                     "Quiet"
                 };
+                string root = RuntimeEnvironmentHelper.IsMono && !RuntimeEnvironmentHelper.IsWindows ? Environment.GetEnvironmentVariable("HOME") : @"c:\";
```

Comment:
Maybe create a util here `Directory.GetDirectoryRoot(Directory.GetCurrentDirectory())`? Or maybe use a temp dir? Or can we use home dir on both platforms? Ideally Windows vs. Mono test code is the same.